### DISCUSSION TITLE
Maybe, rename this repo to 'spot' and write in README not 'SimploTask (aka spot)' but 'spot (aka SimploTask)'

### DISCUSSION THREAD

**Discussion by bessarabov** [umputun/spot#32] [General]
The command line utility is called `spot` and I think that this is a very good name (short, easy to say and to type).

But currentlty the repo where the `spot` is developed is not called `spot`, but it is called `simplotask` ( the name I don't like at all)

I think that this is a generally a good practice to name the repo the same way the main application is called. Now this leads to some confusion

 — Where can I get `spot` binary
 — Oh, it is in `simplotask` repo
 
I understand that you've bought a domain for this — https://simplotask.com, so this makes this change more difficult.

**Comment by umputun** [comment]
Yeah, I agree. I have invented the name simplotask primarily based on available domains. To shorten the utility name, I initially called it "spt" but later figured out what `spot` is much nicer.

I'll see what domain I can get for a spot; sure it won't be spot.com :) Maybe something like spot-tasks or smth like this. Open for suggestions

**Comment by bessarabov** [comment]
I was thinking about `spotapp.com`, but it is already taken. I'm surprised but `spot-scm.com` is also taken (scm for  `Software configuration management`). I've checked several more ideas, and all of that domains are already occupied (I had no idea that `.com` domains are so heavily bought out)

**Comment by umputun** [comment]
what you thing about spot-deploy.com ? or spot-task.com ? or task-spot.com

this domains with dash are available

**Comment by bessarabov** [comment]
Hmm... Frankly speaking I don't like them all

* spot-deploy.com — the deploy is the essential part of spot, but I'm not sure that this will be the main thing for all users (for example I've already using spot in one my real-life task where I need to sync files to server)
* spot-task.com — by looking only to domain name one can think that this is some todo list / task manager
* task-spot.com — I think that it is better to have the main name of the product to be the first before the dash

From those 3 domains I like the best `spot-task.com`, but it is not perfect (but probably it is good enough)

**Comment by bessarabov** [comment]
Thank you for renaming this repo to `spot`!

I'm closing this conversation now, if needed, please reopen it (actually I'm not a power user of github discussions, I'm not 100% sure that it is possible to reopen closed discussions, but I assume that this is possible)

**Reply by bessarabov** [reply]
(haha :) after closing the conversation I see that there is a button "Reopen the discussion", so it is definitely possible)
